sklearn/decomposition/_dict_learning.py

```diff
@@ -46,14 +46,14 @@ def _sparse_encode(X, dictionary, gram, cov=None, algorithm='lasso_lars',
         The dictionary matrix against which to solve the sparse coding of
         the data. Some of the algorithms assume normalized rows.
 
-    gram : None | array, shape=(n_components, n_components)
+    gram : None | array of shape (n_components, n_components)
         Precomputed Gram matrix, dictionary * dictionary'
         gram can be None if method is 'threshold'.
 
-    cov : array, shape=(n_components, n_samples)
+    cov : array of shape (n_components, n_samples), default=None
         Precomputed covariance, dictionary * X'
 
-    algorithm : {'lasso_lars', 'lasso_cd', 'lars', 'omp', 'threshold'}
+    algorithm : {'lasso_lars', 'lasso_cd', 'lars', 'omp', 'threshold'}, default='lasso_lars'
```

Comment:
Thanks for the PR! You need to break long lines with ``\``, see the linting errors.